app/Http/Controllers/SettingsController.php

```diff
@@ -46,22 +47,22 @@ public function storeAppName(Request $request)
     }
 
 
-    public function storeAppLogo(Request $request)
-    {
-        $request->validate([
-            'app_logo' => 'required|image|mimes:jpg,png,jpeg,gif,svg|max:2048',
-            'label' => 'required'
-        ]);
-        $path = 'storage/' . Storage::disk('public')->putFileAs(
-            'logos',
-                $request->file('app_logo'),
-                now()->format('d-m-y-H-i-s').'.'.$request->file('app_logo')->extension());
```

Comment:
hey is there an already existing format you could use? carbon has a lot of them prebuit -> toDateString, toDatetimeString, toISO...String.